signer/tests/integration/block_notifier.rs

```diff
@@ -0,0 +1,29 @@
+use futures::StreamExt;
+use signer::block_notifier::{BlockNotifier, ElectrumBlockNotifier};
+use tracing::{error, info, warn, Level};
+use tracing_subscriber;
+
+#[ignore]
+#[tokio::test]
+async fn test_electrum_block_notifier_integration() {
+    // Initialize logging
+    tracing_subscriber::fmt().with_max_level(Level::INFO).init();
+
+    // Create the notifier
+    let notifier = ElectrumBlockNotifier::from_config().unwrap();
```

Comment:
I'd like the test configuration to be explicit and not taken from the system config if possible.